core/transport/src/main/proto/grpc/file_system_master.proto

```diff
@@ -95,6 +95,9 @@ message OpenFilePOptions {
   optional int32 maxUfsReadConcurrency = 2;
   optional FileSystemMasterCommonPOptions commonOptions = 3;
   optional bool updateLastAccessTime = 4 [default = true];
+  // If specified and the blocks are not cached in any worker,
+  // the data will be read and cached to the certain worker.
+  optional grpc.WorkerNetAddress ufsReadWorkerLocation = 15;
```

Comment:
why `ufsReadWorkerLocation` instead of `readWorkerLocation`?

Reply:
according to @dbw9580 , we are not able to specify which worker we read a block from, if the data is cached in a worker. 
This field, as mentioned in comment, is only to specify a worker to load the data from UFS if the data does not exist in alluxio